src/constants.ts

```diff
@@ -8,3 +8,6 @@ export const DEFAULT_VIDEO_CONSTRAINTS: MediaStreamConstraints['video'] = {
 export const SELECTED_AUDIO_INPUT_KEY = 'TwilioVideoApp-selectedAudioInput';
 export const SELECTED_AUDIO_OUTPUT_KEY = 'TwilioVideoApp-selectedAudioOutput';
 export const SELECTED_VIDEO_INPUT_KEY = 'TwilioVideoApp-selectedVideoInput';
+
+// This is used to store the current background settings in localStorage
+export const BG_SETTINGS_KEY = 'bgsettings5';
```

Comment:
Just use `bgsettings` instead of `bgsettings5` 😄 
I put 5 in there in the prototype code because I wanted to bust the cache.

Reply:
What about `export const SELECTED_BACKGROUND_SETTINGS_KEY = 'TwilioVideoApp-selectedBackgroundSettings'`? Just to be more descriptive and to have some consistency with the other keys. With a more descriptive variable name, we won't need a comment to describe the variable.